themes/hummingbird/hooks-hummingbird-theme.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Hooks on Hummingbird Theme
+weight: 3
+---
+
+# Hooks on Hummingbird Theme
+
+Hummingbird theme introduced a brand new UI for the PrestaShop front office. 
+
+A visual map of hook locations was created by the Design team at PrestaShop. 
+
+{{% notice note %}}
+This page integrates Figma iframes, it may require heavy resources and cause performance issues on slower machines.
```

Comment:
```suggestion
This page embeds Figma views in an iframe, which may slow down your device. To avoid this, please click on the links in the article to open the designs in a new tab.
```